partials/archive-exhibition-item.php

```diff
@@ -0,0 +1,29 @@
+<?php
+$artists = wp_get_post_terms($post->ID, 'artist');
+$title = get_post_meta($post->ID, '_igv_exhibition_title', true);
+$start = get_post_meta($post->ID, '_igv_exhibition_start', true);
+$end = get_post_meta($post->ID, '_igv_exhibition_end', true);
+$location = wp_get_post_terms($post->ID, 'location');
+?>
+<article <?php post_class('grid-item item-s-12 grid-row no-gutter'); ?> id="post-<?php the_ID(); ?>">
+  <a href="<?php the_permalink() ?>" class="grid-item item-s-12 grid-row no-gutter">
```

Comment:
I wish there was a better way to do this markup here but I can't work it out. Just seems sad that the `<a>` is required but the `<article>` is actually not required. Perhaps we can just drop the article as it isn't a full article so not sure semantically there is anything wrong with just an anchor